docs/contributing-to-analyses/analysis-modules/script-structure.md

```diff
@@ -112,13 +117,19 @@ parser.add_argument(
     # Description of the argument
     help = "path to input AnnData file for analysis"
 )
+## define any additional arguments here ##
+
+# parse the arguments into an object
+args = parser.parse_args()
 ```
 
-In the above example, you can access the parsed argument as `parser.input_anndata_file`.
+In the above example, you can access the parsed argument as `args.input_anndata_file`.
+
 
 
 #### Additional Python tips
 
-- You may wish to enclose your opening comments inside a [triple-quote block](https://www.geeksforgeeks.org/triple-quotes-in-python/).
-- Following Python convention, you may also want to define [a `main()` function](https://realpython.com/python-main-function/) to handle most of the script work.
+- Please use [docstrings](https://www.geeksforgeeks.org/python-docstrings/) when adding documentation to any custom functions you write.
```

Comment:
Make this a bit more direct? Or maybe something in between. I mostly don't want it to sound like documentation is optional.
```suggestion
- Include [docstrings](https://www.geeksforgeeks.org/python-docstrings/) for all custom functions you write.
```